Makefile

```diff
@@ -36,7 +36,8 @@ help: Makefile
 install-dep: install-dependencies
 
 install-dependencies:
-	pip2 install --upgrade $(DEVPKGS) || pip install --upgrade $(DEVPKGS)
+	pip install --upgrade $(DEVPKGS)
+	pip install git+https://github.com/ged-lab/screed.git@py3
```

Comment:
This will go away when the py3 branch is merged into screed master and released.